docs/proposals/rename-group.md

```diff
@@ -0,0 +1,156 @@
+- Various annotations and finalizers set by TopoLVM
+- Various resource settings including CSIDriver resource
+- CRD name
+
+Also, by setting the variable of helm chart, it can be used as it is at `topolvm.cybozu.com` as before.
+In addition, we will guide you through the migration procedure for users who wish to migrate their group names.
+
+## Design details
+
+### Changes to topolvm.io
+
+Change the group name `topolvm.cybozu.com` used below to `topolvm.io`.
+
+#### constants.go
+
+constant name | value
+--- | ---
+CapacityKeyPrefix | capacity.topolvm.cybozu.com/
+CapacityResource | topolvm.cybozu.com/capacity
+PluginName | topolvm.cybozu.com
+TopologyNodeKey | topology.topolvm.cybozu.com/node
+DeviceClassKey | topolvm.cybozu.com/device-class
+ResizeRequestedAtKey | topolvm.cybozu.com/resize-requested-at
+LogicalVolumeFinalizer | topolvm.cybozu.com/logicalvolume
+NodeFinalizer | topolvm.cybozu.com/node
+PVCFinalizer | topolvm.cybozu.com/pvc
+
+#### CRD
+
+Change from `logicalvolumes.topolvm.cybozu.com` to` logicalvolumes.topolvm.io`.
+
+#### Resources
+
+##### CSIDriver
+
+Rename the `topolvm.cybozu.com` CSIDriver resource to `topolvm.io`
+
+##### StorageClass
+
+Modify each StorageClass resource in TopoLVM as follows:
+
+- Change the value of the `provisioner` field from `topolvm.cybozu.com` to `topolvm.io`
+- Rename the key name `topolvm.cybozu.com/device-class` in the` parameters` field to `topolvm.io/device-class`
+
+**Since it is not possible to change the StorageClass resource, it is necessary to recreate the resource when migrating the group name.**
+
+##### LogicalVolume
+
+Modify each LogicalVolume resources in TopoLVM as follows:
+
+- Change the CRD to use from `logicalvolumes.topolvm.cybozu.com` to` logicalvolumes.topolvm.io`
+- Change the `topolvm.cybozu.com/resize-requested-at` annotation to` topolvm.io/resize-requested-at`
+- Change the `topolvm.cybozu.com/logicalvolume` finalizer to` topolvm.io/logicalvolume`
+
+**Since the CRD itself changes for the Logical Volume resource, it is necessary to delete the old CRD resource and create a new CRD resource when migrating the group name. Also, in order to prevent the deletion of the actual LVM volume due to the logical volume migration, it is recommended to migrate with the TopoLVM components stopped.**
+
+##### Node
+
+Make the following changes for each Node resources
+
+- Change the `topolvm.cybozu.com/node` finalizer to` topolvm.io/node`
+
+##### PersistentVolumeClaim
+
+Modify each PersistentVolumeClaim resources in TopoLVM as follows:
+
+- Change the value of the `volume.beta.kubernetes.io/storage-provisioner` annotation to` topolvm.io`
+- Change the value of the `volume.kubernetes.io/storage-provisioner` annotation to` topolvm.io`
+- Change the `topolvm.cybozu.com/pvc` finalizer to` topolvm.io/pvc`
+
+##### PersistentVolume
+
+Modify each PersistentVolume resource in TopoLVM as follows:
+
+- Change the value of the `pv.kubernetes.io/provisioned-by` annotation to` topolvm.io`
```

Comment:
I could not find files where these changes are applied. Could you tell me which files were modified?

Reply:
this is here: https://github.com/kubernetes/kubernetes/blob/a7b1fcba40784af64c949b28e7a2ca03433a3a75/pkg/controller/volume/persistentvolume/pv_controller.go#L1662